diagnosticsModule/Public/Join-DiagnosticsFiles.ps1

```diff
@@ -0,0 +1,106 @@
+<#
+.SYNOPSIS
+Merges one or more diagnostics output files into a single merged file named MergedDiagnosticsFile.json.  This merged file is used with the Diagnostics Analyzer Tool on the AD FS Help website (https://adfshelp.microsoft.com/DiagnosticsAnalyzer).
+
+.DESCRIPTION
+The Join-DiagnosticsFiles cmdlet is used to merge one or more diagnostics output files into a single file.  You can then upload this merged file to AD FS Help and view the farm health and configuration holistically.
+
+.PARAMETER FilePath
+The location of the output files that will be merged
+
+.EXAMPLE
+Join-DiagnosticsFiles c:\output
+Merge the diagnostics files located in the c:\output folder
+
+.EXAMPLE
+Join-DiagnosticsFiles .
+Merge the diagnostics files located in the current folder
+#>
+Function Join-DiagnosticsFiles
+{
+    [CmdletBinding()]
+    Param
+    (
+        [string] $FilePath = $null
```

Comment:
we shouldn't make the filepath mandatory, if it is not provided we should try to enumerate the diagnostics files in the current working directory

Reply:
Agree